nix/build-dapp-package.nix

```diff
@@ -47,6 +48,7 @@ in
           '')
           passthru.libPaths;
       buildPhase = ''
+        source ${pkgs.writeText "dapprc" dapprc}
```

Comment:
@asymmetric do you know how I could actually write this file as a `.dapprc` into the source directory used for the nix build? I always ended up running into permissions errors.

Reply:
Why do you want the dapprc to be in the source directory? You need it to be somewhere at runtime right? So it should be in the final store path, no?

Or do you actually need it at build-time because it changes the build behaviour?

Reply:
build time only

Reply:
Although now you mention it it should be in the current working directory instead of the $src dir so I think I was just trying to put it in the wrong place...